course/steps/06_graphql-implementation/en.md

```diff
@@ -0,0 +1,106 @@
+# GraphQL: Retrieving data from Masterdata
+
+## Introduction
+
+Now that we have updated the products count, we need to retrieve the _top n_ most viewed products. We can, then, use Masterdata to retrieve the product page view data and sort by the _count_ field. We can also limit the length of retrieved products, creating a customized size rank of most visited products.
+
+## GraphQL
+
+To get these product page views, we will use [GraphQL](https://graphql.org/), the technology used by VTEX IO for data fetching, to implement a query to Masterdata. GraphQL allows us to implement queries in a simple and easy way, specifying the data you want to retrieve. This makes your API reliable, since GraphQL controlls the data fetched instead of the server itself.
```

Comment:
Enforça aqui em negrito que toda vez que for criar algo pra o frontend consumit, tem que ser em GraphQL

Reply:
Maybe: It's also the only possible way to create an interface between services and front end applications?